CHANGELOG.md

```diff
@@ -76,6 +76,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1504](https://github.com/realm/SwiftLint/issues/1504)
 
+* Add `multiline_parameters` opt-in rule that warns to either keep
+  all the parameters of a method or function on the same line, or one per line.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes